ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -373,6 +362,24 @@ private[apiserver] final class ApiUserManagementService(
           )
       }
 
+  // Here we check if user exists and actually belongs to the requested Identity Provider
+  // Alternatively, identity_provider_id could be part of the compound unique key within user database.
```

Comment:
There's probably no need to make `identity_provider_id` part of a database key to have this check implemented in the persistence layer. Implementing it there would have two benefits:
- the check would be done within a db transaction,
- we would avoid doing the additional `getUser` calls in this method.

Reply:
Additionally this pretty much would create per idp username namespaces in turn would lead to cleaner errors in cases like `creating a user 'alice' in idp 'foo' when an 'alice' already exists in 'bar'`

Reply:
As agreed offline - we dont do it